test/blackbox-tests/test-cases/pkg/gh10741.t

```diff
@@ -0,0 +1,13 @@
+The issue with the oneshot webserver used for testing
+package management, where attempting to send a large file causes the
+connection to be reset before the transfer completes, is now fixed.
+
+Make a large file:
+  $ dd if=/dev/zero of=./large-file iflag=fullblock,count_bytes status=none count=64M
+
+Run the oneshot webserver to serve the large file:
+  $ webserver_oneshot --content-file ./large-file --port-file port.txt &
+  $ until test -f port.txt ; do sleep 0.1; done
```

Comment:
I'm not sure to understand: why you need this file?

Reply:
Which file ?

Reply:
If it is the test file `gh10741.t`, it coming from https://github.com/ocaml/dune/pull/10742 which reproduced the issue.

Reply:
I think we should merge #10742 first and rebase the fix on it.

Reply:
For sure, we could go this.

Reply:
Sorry, my question wasn't well written and clear. Why do you need to test the presence of the `port.txt` file in the test?
```
$ until test -f port.txt ; do sleep 0.1; done
```
If it is executed sequentially, we shouldn't have to wait, no?

Reply:
The server is started in the background (`&`) thus by the time the next line runs, the file might not yet be created, so the next line waits until the server is up and creates the port file (which is its way of signalling "I'm ready to accept a connection").

The server can't be started in the foreground, as then the test never terminates.